src/ebookmaker/writers/__init__.py

```diff
@@ -61,6 +62,55 @@ def write_with_crlf(filename, bytes_):
         with open(filename, 'wb') as fp:
             fp.write(bytes_)
 
+    @staticmethod
+    def writec_with_crlf(filename, bytes_):
+        t = bytes_.splitlines()
+        u = []
+        for item in t:  # convert to strings
+            u.append(item.decode('utf-8'))
+
+        u2 = []
+        for line in u:
+            if len(line) > 75 and not "Most recently updated" in line:
+                w = textwrap.wrap(line, width=72)
```

Comment:
As this code appears to be invoked on the entire file, it may change the wrapping from the source file, not desired  when the preparer uses the lines to replicate the wrapping in the source.